packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -122,6 +125,7 @@ export declare namespace UpdateRelease {
       name: string;
       // When editing a release, scheduledAt always need to be explicitly sended, so it can be null to unschedule it
       scheduledAt?: Date | null;
```

Comment:
I am not sure why you need to define `scheduledAt`, `timezone` optional(undefined) even when updating. 
This creates alot confusion on front and I end of with `scheduledAt: Date | null | undefined`, But I feel null is sufficient even when you create a release, introducing `undefined` is not needed.

Reply:
This is like this because, currently, we don't have the front part so right now we are using the query without scheduledAt, you can safely remove the undefined on your PR for this 👍